.github/workflows/release.yml

```diff
@@ -86,3 +86,44 @@ jobs:
           asset_path: kani-${{ needs.Release.outputs.version }}-${{ matrix.target }}.tar.gz
           asset_name: kani-${{ needs.Release.outputs.version }}-${{ matrix.target }}.tar.gz
           asset_content_type: application/gzip
+
+  Package-Docker:
+    name: 'Package Docker'
+    needs: Release
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: write
+    strategy:
+      os: ubuntu-20.04
+      target: x86_64-unknown-linux-gnu
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Kani Dependencies
+        uses: ./.github/actions/setup
+        with:
+          os: ubuntu-20.04
+
+      - name: 'Build release bundle'
+        run: |
+          cargo run -p make-kani-release -- ${{ needs.Release.outputs.version }}
+
+      - name: 'Login to GitHub Container Registry'
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: model-checking
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          registry: ghcr.io
+          context: .
+          file: scripts/ci/Dockerfile.bundle-test-ubuntu-20-04
```

Comment:
If we're going to publish a docker image should should be creating one specific to running Kani.

For instance, it should not have unnecessary dependencies or anything else shoveled into it.

I don't think our test script qualifies. At the very least I believe it shovels our whole test suite in there.

Reply:
[scripts/ci/Dockerfile.bundle-release-20-04](https://github.com/model-checking/kani/pull/1595#diff-f311bab8b2fd59efa0ac86482bd26cbe00a5df6087d76f25c529d48facb8764c)

I did a few tricks to purge unnecessary things from the image but not much luck. Biggest gain came from purging docs, but the bulk of the files were necessary static objects from Rust compiler internals.

| Image    | Size   |
|-----------------|
| Naive   | 2.89GB |
| Cleaned | 2.29GB  |